### PR TITLE
Angular update

### PR DESCRIPTION
**Pull request by fat23cat**
Last Angular and Angular Material updates.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
 # install angular in a separate cached layer
 RUN mkdir -p /ddp-workspace/projects
-ENV ANGULAR_CLI @angular/cli@8.2.0
+ENV ANGULAR_CLI @angular/cli@8.3.21
 RUN npm install -g $ANGULAR_CLI
```
Comment (MocanaAtBroad) [inline Dockerfile:34]: @zyme We still using Dockerfile s for deployment or have we moved to using Google App Engine configurations?

**Review by MocanaAtBroad — COMMENTED** [review]
Changes look good `ng serve` and prod builds I attempted worked without a problem. Not sure if we still using Dockerfile so have asked Zim to chime in.

File: Dockerfile (hunk 1 above)
Comment (fat23cat) [reply Dockerfile:34]: @zyme, can you help us? ^^

Comment (zyme) [reply Dockerfile:34]: Sorry for the delay!  Yes, we are still using Docker for our deploys--hopefully not too much longer, but we will need to keep doing these sorts of changes for now.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**